### ISSUE TITLE
Rsnapshotconfig throws a 500 internal server error

### ISSUE THREAD

**Issue by xezpeleta** [elkarbackup/elkarbackup#323]
The URL to get the Rsnapshot config `/client/1/job/1/config` is not working.

prod.log:

```
[2018-10-07 11:01:54] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\UndefinedMethodException: "Attempted to call an undefined method named "getEffectiveDir" of class "Binovo\ElkarBackupBundle\Entity\Job"." at /app/elkarbackup/src/Binovo/ElkarBackupBundle/Controller/DefaultController.php line 934 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\UndefinedMethodException(code: 0): Attempted to call an undefined method named \"getEffectiveDir\" of class \"Binovo\\ElkarBackupBundle\\Entity\\Job\". at /app/elkarbackup/src/Binovo/ElkarBackupBundle/Controller/DefaultController.php:934)"} []
```

**Comment by xezpeleta** [comment]
It will be fixed in the upcoming version (v1.3.1)

**Comment by xezpeleta** [comment]
This bug has been fixed in v.1.3.1
